src/System.Private.CoreLib/shared/System/Number.BigInteger.cs

```diff
@@ -410,12 +844,12 @@ public static void Multiply(ref BigInteger lhs, ref BigInteger rhs, ref BigInteg
                 }
             }
 
-            public static void Pow10(uint exponent, ref BigInteger result)
+            public static void Pow10(uint exponent, out BigInteger result)
```

Comment:
Probably some room for improvement here as well, calculating some of the larger powers of 10 can be "expensive".